packages/typespec-python/test/azure/generated/azurecore-lro-rpc/azurecore/lro/rpc/models/__init__.py

```diff
@@ -6,24 +6,15 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from ._models import Error
 from ._models import GenerationOptions
-from ._models import GenerationResponse
 from ._models import GenerationResult
-from ._models import InnerError
-
-from ._enums import OperationState
 from ._patch import __all__ as _patch_all
 from ._patch import *  # pylint: disable=unused-wildcard-import
 from ._patch import patch_sdk as _patch_sdk
 
 __all__ = [
-    "Error",
     "GenerationOptions",
-    "GenerationResponse",
     "GenerationResult",
-    "InnerError",
-    "OperationState",
```

Comment:
Is the change related with typespec-azure-core or TCGC?

Reply:
it is from this pr: https://github.com/Azure/typespec-azure/pull/962, i refined the logic for core model filtering.